spec/services/billable_metrics/prorated_aggregations/unique_count_service_spec.rb

```diff
@@ -358,24 +360,20 @@
         let(:previous_event) do
           create(
             :event,
+            organization_id: organization.id,
             code: billable_metric.code,
-            customer:,
-            subscription:,
+            external_customer_id: customer.external_id,
+            external_subscription_id: subscription.external_id,
             timestamp: from_datetime + 5.days,
-            quantified_event: previous_quantified_event,
             properties: {
-              unique_id: '000',
-            },
-            metadata: {
-              current_aggregation: '7',
-              max_aggregation: '7',
-              max_aggregation_with_proration: '5.8',
-            },
+              unique_id: previous_quantified_event.external_id,
+            }
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)